feat: manufacturing documentation for v13 (#74)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/manufacturing/manufacturing-dashboard.md

<!-- add-breadcrumbs -->

> #Manufacturing Dashboard

Reviewer: ```suggestion
# Manufacturing Dashboard
```

To access the Manufacturing Dashboard, go to:

> Home > Customization > Dashboard > Manufacturing

<img class="screenshot" alt="Manufacturing Dashboard" src="{{docs_base_url}}/assets/img/manufacturing/manufacturing-dashboard.png">

## Number Cards

<img class="screenshot" alt="Number Cards" src="{{docs_base_url}}/assets/img/manufacturing/manufacturing-number-cards.png">

> 1. Monthly Total Work Order :- The user will get the total count of Work Orders which are in Open, In Progress, and Completed state. The creation of the Work Orders is one month back from the current date will be displayed in the number card.
> 1. Monthly Completed Work Order :- The user will get the total count of Work Orders which are in the Completed state. The creation of the Work Orders is one month back from the current date will be displayed in the number card.
> 1. Ongoing Job Card :- The user will get the total count of Job Cards that are not in the Completed state.
> 1. Monthly Quality Inspection :- The user will get the total count of Quality Inspection records which are in the Submitted state. The creation of the work orders is one month back from the current date will be displayed in the number card.

Reviewer: ```suggestion
1. **Monthly Total Work Order**: You will get the total count of Work Orders which are in Open, In Progress, and Completed state. The Work Orders created one month before the current date will be displayed in the number card.
1. **Monthly Completed Work Order**: The user will get the total count of Work Orders which are in the Completed state. The Work Orders created one month before the current date will be displayed in the number card.
1. **Ongoing Job Card**: The user will get the total count of Job Cards that are not in the Completed state.
1. **Monthly Quality Inspection**: The user will get the total count of Quality Inspection records which are in the Submitted state. The Work Orders created one month before the current date will be displayed in the number card.
```

## Dashboard

> 

Reviewer: ```suggestion

You can see some predefined graph reports in the dashboard when you open the Manufacturing module. These reports are fully customizable, you can choose what to show or hide and also configure the metrics on which the reports are shown.

```

<img class="screenshot" alt="Produced Quantity & Completed Operation" src="{{docs_base_url}}/assets/img/manufacturing/manufacturing-dashboard-2.png">

### Produced Quantity

> The chart will give the information about the total number of quantity produced (using work order) in the last year based on Quarterly. Users can also view the chart data based on Daily, Weekly, Monthly, Quarterly, Yearly basis.

Reviewer: ```suggestion
The chart will give the information about the total quantity produced (using Work Order) in the last year on a Quarterly basis. Users can also view the chart data based on Daily, Weekly, Monthly, Yearly basis.
```

### Completed Operation

> The chart will give information about the total number of operations completed in the last year based on Quarterly. Users can also view the chart data based on Daily, Weekly, Monthly, Quarterly, Yearly basis.

Reviewer: ```suggestion
The chart will give information about the total number of operations completed in the last year on Quarterly basis. Users can also view the chart data based on Daily, Weekly, Monthly, Yearly basis.
```

<hr>

<img class="screenshot" alt="Work Order & Quality Inspction Analysis" src="{{docs_base_url}}/assets/img/manufacturing/manufacturing-dashboard-3.png">

### Work Order Analysis

> The chart will give the information about the number of Work Orders based on Not Started, In Process, Stopped, Completed status. The chart type is a Donut and it will provide the information based on the last year's work order data. Users can also change the date range by clicking on the Filter button.

Reviewer: ```suggestion
This chart will give the information about the number of Work Orders based on Not Started, In Process, Stopped, Completed statuses. This Donut chart will provide the information based on the last year's Work Order data. Your can also change the date range by clicking on the **Filter** button.
```

### Quality Inspction Analysis

> The chart will give information about the number of Quality Inspections based on Accepted and Rejected status. The chart type is a donut and it will provide the information based on the last year's quality inspection data. Users can change the date range by clicking on the Filter button.

Reviewer: ```suggestion
This chart will give information about the number of Quality Inspections based on Accepted and Rejected status. The chart type is a donut and it will provide the information based on the last year's quality inspection data. Your can change the date range by clicking on the **Filter** button.
```

<hr>

<img class="screenshot" alt="Pending Work Order & Downtime Analysis" src="{{docs_base_url}}/assets/img/manufacturing/manufacturing-dashboard-4.png">

> ### Pending Work Order

Reviewer: ```suggestion
### Pending Work Orders
```
it should be "Pending Work Orders" not "Pending Work Order"!!

The chart will give information about the number of Work Orders that are pending based on the aging days. The number of aging days is calculated based on the difference between the current date and planned start / actual start date days.

### Last Month Downtime Analysis

> The chart will give information about the total number of minutes the Machine was not working in the last month. This helps the operator to know which machine has not performed well and requires maintenance.

Reviewer: ```suggestion
The chart will give information about the total number of minutes a machine was not working in the last month. This helps the operator to know which machine has not performed well and requires maintenance.
```

<hr>

<img class="screenshot" alt="Work Order Quantity Analysis" src="{{docs_base_url}}/assets/img/manufacturing/manufacturing-dashboard-5.png">

### Work Order Quantity Analysis

The chart will give information about the total number of quantities that are Pending and Completed based on the Work Orders every month for the last year.

<hr>

<img class="screenshot" alt="Job Card Analysis" src="{{docs_base_url}}/assets/img/manufacturing/manufacturing-dashboard-6.png">

### Job Card Analysis

> The chart will give information about the total number of Job Card which is in Pending and Completed state every month for the last year.

Reviewer: ```suggestion
The chart will give information about the total number of Job Cards which are in the Pending and Completed states every month for the last year.
```
